### PR TITLE
add Helper class

### PR DESCRIPTION
**Pull request by scorpion01**

### CODE REVIEW

File: app/src/main/java/com/byteshaft/iblinklednotification/CallBilnker.java [hunk 1]
```diff
@@ -3,19 +3,19 @@
...
 public class CallBilnker extends BroadcastReceiver implements CameraStateChangeListener {
     private com.byteshaft.ezflashlight.Flashlight mFlashlight;
 
     @Override
     public void onReceive(Context context, Intent intent) {
+        Helpers mHlepers = new Helpers(context);
+
         Toast.makeText(context.getApplicationContext(), "Received", Toast.LENGTH_SHORT).show();
-        if (isSmsBlinkingEnabled(context)) {
+        if (mHlepers.isSmsBlinkingEnabled(context)) {
```
Comment (om26er) [inline app/src/main/java/com/byteshaft/iblinklednotification/CallBilnker.java]: mHelpers is not a member variable, only member variables should start with 'm'. Also you should not pass 'context' directly into the method rather call 'context.getApplicationContext()'

File: app/src/main/java/com/byteshaft/iblinklednotification/Helpers.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import android.content.ContextWrapper;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+
+public class Helpers extends ContextWrapper{
+    public Helpers(Context base) {
+        super(base);
+    }
+
+    boolean isSmsBlinkingEnabled(Context context) {
```
Comment (om26er) [inline app/src/main/java/com/byteshaft/iblinklednotification/Helpers.java]: This method should no longer take Context as parameter. This class extends ContextWrapper so the whole class has it and as a result every method can use it.

File: app/src/main/java/com/byteshaft/iblinklednotification/SmsBlinking.java [hunk 3]
```diff
@@ -15,7 +13,8 @@
 
     @Override
     public void onReceive(Context context, Intent intent) {
-        if (isSmsBlinkingEnabled(context)) {
+        Helpers mHelpers = new Helpers(context);
+        if (mHelpers.isSmsBlinkingEnabled(context)) {
```
Comment (om26er) [inline app/src/main/java/com/byteshaft/iblinklednotification/SmsBlinking.java]: Same. mHelpers to helpers.

**Comment by scorpion01** [comment]
fixed all suggestions

**Pull request merged**